apps/api/v2/src/modules/slots/controllers/slots.controller.e2e-spec.ts

```diff
@@ -0,0 +1,28 @@
+import { bootstrap } from "@/app";
```

Comment:
This file supposed to be in another branch?

Reply:
Thats... interesting. Yea.